pkg/agent/plugin/workloadattestor/docker/docker.go

```diff
@@ -123,8 +123,8 @@ func (p *Plugin) Configure(ctx context.Context, req *configv1.ConfigureRequest)
 		return nil, err
 	}
 
-	if err := validateOS(config); err != nil {
-		return nil, err
+	if len(config.UnusedKeys) > 0 {
+		return nil, status.Errorf(codes.InvalidArgument, "unknown configuration detected: %s", strings.Join(config.UnusedKeys, ","))
```

Comment:
If we're going to introduce this, we probably need to warn for a deprecation cycle. This can break existing deployments.